src/views/blocks/parts/details.php

```diff
@@ -73,55 +74,65 @@
 		if ( tribe_event_is_all_day() && tribe_event_is_multiday() ) :
 			?>
 
-			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_date ); ?> </abbr>
```

Comment:
* :no_entry_sign: The $text arg must be a single string literal, not "$start_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$start_date".

<sub>hash: 5c1e0d5b3d21956f7131a0c7ad183a84</sub>